great_expectations/core/usage_statistics/payload_builder.py

```diff
@@ -83,6 +85,7 @@ def build_envelope(self, message: dict) -> dict:
         message["data_context_instance_id"] = self._data_context_instance_id
 
         message["mac_address"] = self._determine_hashed_mac_address()
+        message["oss_id"] = str(self._oss_id) if self._oss_id else None
```

Comment:
Schema validation gets made if we don't explicitly convert to `str` - figure it was best to keep this a rich type up until the very last minute

Reply:
`non-blocking`
If we were using a pydantic data model for the payload it could stay a rich type as long as we are within python and use the pydantic `.json()` method to serialize the request body.

Reply:
Hmm I like it but there’s a very good chance we toss all this work aside in favor of a more robust 3rd party solution - mind if we table this schema tightening until we’re aligned on a future direction for usage stats?

Reply:
yeah if this pans out for us and it increases our confidence in our ability to reliably count unique entities, then we're going to proceed with 3rd party framework development in parallel. old system will likely stay in place for awhile until we can migrate everything to new system.